### ISSUE TITLE
Evaluate & specify FMS Backend - Frontend interface

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#19]
## User story
1. As a SD
2. I want to brainstorm on the interface between the backend and frontend of the fleet management software
3. So that have a better idea on how these components communicate.
[VDA5050-V2_0_0-DE.pdf]
(https://github.com/amosproj/amos2022ss03-turtlebot-fleet-management/files/8667729/VDA5050-V2_0_0-DE.pdf)

## Acceptance criteria
* Good understanding of the interface between specified components. 
* Documenting a standard on how the components communicate, aka create API docs.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by cat24max** [comment]
This has nothing to do with VDA5050.

API docs already exist: https://github.com/amosproj/amos2022ss03-turtlebot-fleet-management/blob/main/FleetMgmnt/WebUI/apidoc.txt

**Comment by UmangBR** [comment]
VDA50550 is relevant to #18 and not #19. Correct me if I'm wrong.

**Comment by cat24max** [comment]
Yes

**Comment by cat24max** [comment]
I would recommend we move this to the product backlog for now. The interface is finished in its current state, but we will 100% have to extend it in the future after the FMS and Robots can communicate.

Right now, there is nothing to be done.

**Comment by UmangBR** [comment]
Note: this issue is an extension of #17
